twisted/internet/base.py

```diff
@@ -443,6 +443,13 @@ class ReactorBase(object):
         and C{reactor.stop}.  This should be replaced with an explicit state
         machine.
 
+    @type _lastSignal: None|C{int}|C{bool}
```

Comment:
You're right.  I simplified it a lot by hard coding the signal numbers, so it works on-or-off Posix systems, no longer has ugly conditional module imports and now it's only a single data time: Int.